src/apps/patron-page/PatronPage.dev.tsx

```diff
@@ -177,7 +179,21 @@ export default {
       control: { type: "text" }
     },
     patronPageStatusSectionReservationsText: {
-      defaultValue: "You can reserve {number} ebooks and {number} audiobooks",
+      defaultValue:
+        "You can reserve @countEbooks ebooks and @countAudiobooks audiobooks",
+      control: { type: "text" }
+    },
+    patronPageStatusSectionOutOfText: {
+      defaultValue: "@this out of @that",
+      control: { type: "text" }
+    },
+    patronPageStatusSectionOutOfAriaLabelAudioBooksText: {
+      defaultValue:
+        "You used @this audiobooks out of you quota of @that audiobooks",
```

Comment:
Why not @audioBooksConsumed and @audioBooksMaximum?
I think it is easier to understand for the administrator trying to understand the meaning of the sentence.